packages/vertexai/src/api.ts

```diff
@@ -45,20 +45,26 @@ declare module '@firebase/component' {
 export function getVertexAI(app: FirebaseApp = getApp()): VertexAI {
   app = getModularInstance(app);
   // Dependencies
-  const vertexProvider: Provider<'vertex'> = _getProvider(app, VERTEX_TYPE);
+  const vertexProvider: Provider<'vertexAI'> = _getProvider(app, VERTEX_TYPE);
```

Comment:
Could change Provider<\`vertexAI\`> to Provider<VERTEX_TYPE> here, I think.

Reply:
Sadly I cannot, Typescript can't turn an actual code string into a string that you use for typing.